gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -191,6 +191,21 @@ public static String getConfigSummary(Settings settings, Device.PerfettoCapabili
                   .addAllMeminfoCounters(Arrays.asList(MEM_COUNTERS));
     }
 
+    boolean large_buffer = false;
+    if (settings.perfettoVulkanCPUTiming) {
+      large_buffer = true;
+      config.addDataSourcesBuilder()
+        .getConfigBuilder()
+          .setName("CPUTiming");
```

Comment:
Should we use the full "VkLayer_CPUTiming" name, so that we don't risk name collisions?
Or maybe "vulkan.CPUTiming".

Reply:
We can do either, it was nice to have all of the `layer names == producer name`, but I'm not too strongly tied.